### PR TITLE
Add Training Log table

### PR DESCRIPTION
**Pull request by jules2689**
cc @richardwu

### CODE REVIEW

File: db/migrate/20160909021628_training_log.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class TrainingLog < ActiveRecord::Migration[5.0]
+  def up
+    create_table :training_logs do |t|
+      t.string :root_domain, unique: true, index: true
```
Comment (jules2689) [inline db/migrate/20160909021628_training_log.rb]: LOL good catch

File: db/migrate/20160909021628_training_log.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      t.string :uri, index: true
```
Comment (richardwu) [inline db/migrate/20160909021628_training_log.rb]: Actually, declaring `unique` here doesn't actually put a `unique` index constraint in the DB.

You'll have to do `add_index :training_logs, :uri, index: true`

**Comment by jules2689** [comment]
GTM? @richardwu

File: db/migrate/20160909021628_training_log.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  def up
+    create_table :training_logs do |t|
+      t.string :root_domain, index: true
+      t.string :uri
+      t.string :trained_status, default: 'untrained', index: true
+      t.timestamps
+    end
+
+    add_index :training_logs, [:root_domain, :trained_status]
+    add_index :training_logs, :uri, unique: true
```
Comment (richardwu) [inline db/migrate/20160909021628_training_log.rb]: 👌

**Comment by richardwu** [comment]
LGTM

**Pull request merged**